### ISSUE TITLE
add images from the web

### ISSUE THREAD

**Issue by jflheureux** [burkmarr/tombiovis#17]
Currently, tombiovis only supports image-local for adding images. Any plans for adding image-web support to add any http(s) URL? It would allow to link to images already stored elsewhere like Flickr, SmugMug, Google Photos...

I am a developer and I could code this feature as it is required for one of my project. Before doing so, I wanted to check with you whether it was already done in your v1.7 branch, already planned, or if there is anything I should know before starting?

Thanks

**Comment by burkmarr** [comment]
Hi Jean-François. I would be interested ti see what you come up with. The work for 1.7 was being done in the branch Architecture1 which I have just merged with master and then deleted. So now the work for 1.7 is taking place in master. The Identikit - which is what we call it now - is undergoing a great deal of development and restructuring this year, so my advice would be to isolate your code as much as possible (e.g. in a separate module) so that it can be re-used without too much pain when other areas of code change. Perhaps do something small first - just as a proof of concept. Thanks for your offer of help!

**Comment by jflheureux** [comment]
Thank you for your reply. You are mentioning a separate module. However, i do not see any documentation related to module support for Identikit. Do you support a module/plug-in/extension architecture?

As for the merge in master, I do not see the commit(s) on GitHub but I do see that the Architecture1 branch is gone. Are you missing a push of your local changes?

**Comment by burkmarr** [comment]
Sorry about that - I hadn't pushed the merge; it's done now. By module I really mean try to maintain separation, where possible, between your code and existing code to make it more flexible and reusable in different scenarios as other underlying Identikit code changes.
